### PR TITLE
Rename SplitBrainProtection API method

### PR DESCRIPTION
**Pull request by petrpleshachkov**
Renamed isMinimumClusterSizeSatisfied() -> hasMinimumSize().

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/splitbrainprotection/impl/SplitBrainProtectionImpl.java [hunk 1]
```diff
@@ -162,7 +162,7 @@ public SplitBrainProtectionConfig getConfig() {
     }
 
     @Override
-    public boolean isMinimumClusterSizeSatisfied() {
+    public boolean hasMinimumSize() {
         return splitBrainProtectionState == SplitBrainProtectionState.MIN_CLUSTER_SIZE_SATISFIED;
```
Comment (ahmetmircik) [inline hazelcast/src/main/java/com/hazelcast/splitbrainprotection/impl/SplitBrainProtectionImpl.java]: minor: This `SplitBrainProtectionState.MIN_CLUSTER_SIZE_SATISFIED` can also be renamed.

**Comment by mmedenjak** [comment]
run-lab-run

**Comment by petrpleshachkov** [comment]
run-lab-run

**Comment by petrpleshachkov** [comment]
Guys, thanks for the review. I am merging the PR.

**Pull request merged**